engine/graphics/src/wscript

```diff
@@ -110,6 +110,9 @@ def build(bld):
 
         elif bld.env.PLATFORM in ('arm64-ios',):
             obj.source.append('vulkan/ios/graphics_vulkan_ios_surface.mm')
+
+            obj.source.append('vulkan/osx/graphics_vulkan_osx.mm')
```

Comment:
A little amazing that we haven't needed it before! ^^

Reply:
But that file is very new (two weeks old), it's just a few very specific functions for mixing vulkan and metal in an extension (you can probably guess which one!) :)